src/app/modules/main/wallet_section/transactions/view.nim

```diff
@@ -84,13 +84,17 @@ QtObject:
     for address in addresses:
       self.setHistoryFetchState(address, isFetching)
 
-  proc switchAccount*(self: View, walletAccount: WalletAccountDto) =
-    if not self.models.hasKey(walletAccount.address):
-      self.models[walletAccount.address] = newModel()
-    self.model = self.models[walletAccount.address]
+  proc setModel*(self: View, address: string) {.slot.} =
```

Comment:
can this be handled under switchAccount itself?

Reply:
switchAccount use a dto not an address

Reply:
I meant switchAccountByAddress